.github/workflows/gha-ci.yml

```diff
@@ -0,0 +1,103 @@
+name: GitHubActions CI tests
+
+on:
+  push:
+    branches: [master, dev, ci]
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15]
+        build_suite: [autoconf]
+        with_coveralls: [no]
+        include:
+          # Do one build with cmake instead of autoconf
+          - os: ubuntu-20.04
+            build_suite: cmake
+            with_coveralls: no
+          # And run one build with gcov and coveralls
+          # (Does currently not work, but neither did it in travis)
+          #- os: ubuntu-20.04
+          #  build_suite: autoconf
+          #  with_coveralls: yes
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+
+      - name: install deps
+        run: |
+          if (echo "${{ matrix.os }}" | grep -qE '^macos-') ; then
+            brew update
+            # freetype, gettext, icuc4 and autoconf are preinstalled
+            # and `brew install` fails if a non-uptodate version is already installed
+            #brew upgrade autoconf freetype gettext icu4c
+            brew install automake boost ffmpeg ffms2 fftw fribidi libass \
+                         lua luarocks m4 wxmac hunspell uchardet
+          else
+            #sudo apt-get update && sudo apt-get upgrade
+            sudo apt-get install -y ${{ matrix.build_suite }} \
+              autopoint build-essential g++ gcc gettext libasound2-dev \
+              libass-dev  libffms2-dev libfftw3-dev libfontconfig1-dev   \
+              libfribidi-dev libgtest-dev libhunspell-dev libicu-dev libopenal-dev       \
+              libpulse-dev libuchardet-dev libwxgtk3.0-gtk3-dev luarocks portaudio19-dev \
+              python3-pip python3-setuptools \
+              pkg-config libgl1-mesa-dev libgl-dev libfreetype6-dev libfontconfig-dev \
+              intltool libboost-all-dev
+            if [ "x${{ matrix.with_coveralls }}" = "xyes" ] ; then
+              sudo pip3 install -U cpp-coveralls
+            fi
+          fi
+
+          sudo luarocks install busted     > /dev/null
+          sudo luarocks install moonscript > /dev/null
+          sudo luarocks install uuid       > /dev/null
+
+          git submodule --quiet init
+          git submodule --quiet update vendor/googletest
```

Comment:
The original Travis script did only do this for autoconf builds, while CMake builds used cmake to compile and install a Travis provided googletest in `/usr/src/googletest`. In case of Linux builds I think it should be possible to install those manually, but I don't know about Homebrew.
If we want to use system googletests, does it even make sense to also use an ancient googletest from `vendor`?

Reply:
`/usr/src/googletest` is provided by the `googletest` package from Ubuntu, not Travis. See https://packages.ubuntu.com/bionic/amd64/googletest/filelist . I have no idea why it is provided in source form without binary.

Finding system provided googletest was not implemented in autoconf build, therefore `vendor/googletest` is always used. It may be possible to [specify its path during make](https://github.com/wangqr/Aegisub/blob/e35c16dc85bb2eb0c18d1c2ad0d42671d58773de/tests/Makefile#L3), but I didn't test that, and it needs to be manually done on different OSs / distos.

Reply:
Restored old behaviour for CMake and Ubuntu-autoconf builds now also use system googletest.